tasks/synopsys-detect-task/ts/runner/DetectScriptConfigurationRunner.ts

```diff
@@ -63,6 +63,7 @@ export class DetectScriptConfigurationRunner extends DetectRunner {
         } catch (error) {
             logger.error(`Unable to connect with ${DetectScriptDownloader.DETECT_DOWNLOAD_URL}`)
             logger.error('This may be a problem with your proxy setup or network.')
+            throw new Error('This may be a problem with your proxy setup or network')
```

Comment:
I'd suggest adding a period to the end to follow the same format as other error messages

Reply:
Also, from versioning stand point for this bug fix, we should make version 8.1.0 release. Lets make sure to update versions in two package.json files, task.json and vss-extension.json to 8.1.0